Fix github action release

# Situation
Currently the build of the Github action fails when installing the package `hub` :
```
Run sudo apt-get -y install hub
Reading package lists...
Building dependency tree...
Reading state information...
The following additional packages will be installed:
  xsel
The following NEW packages will be installed:
  hub xsel
0 upgraded, 2 newly installed, 0 to remove and 11 not upgraded.
Need to get 2449 kB of archives.
After this operation, 6886 kB of additional disk space will be used.
Get:1 file:/etc/apt/apt-mirrors.txt Mirrorlist [144 B]
Ign:2 http://azure.archive.ubuntu.com/ubuntu noble-updates/universe amd64 hub amd64 2.14.2~ds1-1ubuntu3.1
Get:3 http://azure.archive.ubuntu.com/ubuntu noble/universe amd64 xsel amd64 1.2.1-1 [20.5 kB]
Ign:2 https://archive.ubuntu.com/ubuntu noble-updates/universe amd64 hub amd64 2.14.2~ds1-1ubuntu3.1
Ign:2 https://security.ubuntu.com/ubuntu noble-updates/universe amd64 hub amd64 2.14.2~ds1-1ubuntu3.1
Err:2 mirror+file:/etc/apt/apt-mirrors.txt noble-updates/universe amd64 hub amd64 2.14.2~ds1-1ubuntu3.1
  404  Not Found [IP: 20.106.104.242 80]
E: Failed to fetch mirror+file:/etc/apt/apt-mirrors.txt/pool/universe/h/hub/hub_2.14.2%7eds1-1ubuntu3.1_amd64.deb  404  Not Found [IP: 20.106.104.242 80]
Fetched 20.5 kB in 1s (16.9 kB/s)
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
Error: Process completed with exit code 100.
```

# Wanted
The build should be fixed.

--- a/.github/workflows/release-github-action.yml
+++ b/.github/workflows/release-github-action.yml
@@ -49,7 +49,9 @@ jobs:
       # Setup + Caching
       # ----------------------
       - name: Install required packages
-        run: sudo apt-get -y install hub
+        run: |
+          sudo apt-get update
+          sudo apt-get -y install hub
 
       - name: Use Node.js
         uses: actions/setup-node@1d0ff469b7ec7b3cb9d8673fde0c81c44821de2a

